### ISSUE TITLE
onRequest and onResponse support?

### ISSUE THREAD

**Issue by WHK102** [OptimalBits/redbird#14]
Have support of onrequest and onresponse triggers? example, need send additional header to server "x-real-ip" with http.req.connection.remoteAddress and restrict access to certains urls by example /admin

**Comment by manast** [comment]
yes this is something that I have planned and is quite high priority, expect it in the next version.

**Comment by WHK102** [comment]
Thank you very much

**Comment by lorneb** [comment]
+1

**Comment by Burnsage** [comment]
Any updates?

**Comment by vorticalbox** [comment]
> Any updates?
onRequest is in the readme however there is no handling for it in the function

https://github.com/OptimalBits/redbird/blob/master/lib/proxy.js#L392
